esignet-countryA.properties

```diff
@@ -0,0 +1,500 @@
+
+mosip.esignet.binding.salt-length=16
+mosip.esignet.binding.audience-id=esignet-binding
+mosip.esignet.binding.key-expire-days=10
+
+## -------------------------------------- Authentication & Authorization -----------------------------------------------
+
+mosip.esignet.security.auth.post-urls={'${server.servlet.path}/client-mgmt/**' : {'SCOPE_add_oidc_client'} , \
+  \ '${server.servlet.path}/system-info/**' : { 'SCOPE_upload_certificate'},\
+  \ '${server.servlet.path}/binding/wallet-binding' : { 'SCOPE_wallet_binding'}, \
+  \ '${server.servlet.path}/binding/binding-otp' : { 'SCOPE_send_binding_otp'}}
+mosip.esignet.security.auth.put-urls={'${server.servlet.path}/client-mgmt/**' : { 'SCOPE_update_oidc_client'} }
+mosip.esignet.security.auth.get-urls={'${server.servlet.path}/system-info/**' : { 'SCOPE_get_certificate'} }
+
+mosip.esignet.security.ignore-csrf-urls=${server.servlet.path}/oidc/**,${server.servlet.path}/oauth/**,\
+  ${server.servlet.path}/actuator/**,/favicon.ico,${server.servlet.path}/error,\
+  ${server.servlet.path}/swagger-ui/**,${server.servlet.path}/v3/api-docs/**,\
+  ${server.servlet.path}/linked-authorization/link-transaction,${server.servlet.path}/linked-authorization/authenticate,\
+  ${server.servlet.path}/linked-authorization/consent,${server.servlet.path}/binding/**,${server.servlet.path}/client-mgmt/**,\
+  ${server.servlet.path}/vci/**,${server.servlet.path}/system-info/**
+
+mosip.esignet.security.ignore-auth-urls=${server.servlet.path}/csrf/**,${server.servlet.path}/authorization/**,\
+  ${server.servlet.path}/linked-authorization/**,${server.servlet.path}/oidc/**,${server.servlet.path}/oauth/**,\
+  ${server.servlet.path}/actuator/**,/favicon.ico,${server.servlet.path}/error,${server.servlet.path}/swagger-ui/**,\
+  ${server.servlet.path}/v3/api-docs/**,${server.servlet.path}/vci/**,${server.servlet.path}/proxyauthorization/**
+
+spring.security.oauth2.resourceserver.jwt.issuer-uri=${keycloak.external.url}/auth/realms/mosip
+spring.security.oauth2.resourceserver.jwt.jwk-set-uri=${keycloak.external.url}/auth/realms/mosip/protocol/openid-connect/certs
+
+##------------------------------------------ Kafka configurations ------------------------------------------------------
+spring.kafka.bootstrap-servers=kafka-0.kafka-headless.${kafka.profile}:${kafka.port},kafka-1.kafka-headless.${kafka.profile}:${kafka.port},kafka-2.kafka-headless.${kafka.profile}:${kafka.port}
+spring.kafka.consumer.group-id=esignet-consumer
+spring.kafka.consumer.enable-auto-commit=true
+#spring.kafka.listener.concurrency=1
+
+mosip.esignet.kafka.linked-session.topic=esignet-linked
+mosip.esignet.kafka.linked-auth-code.topic=esignet-consented
+
+## ------------------------------------------- Integrations ------------------------------------------------------------
+
+mosip.esignet.integration.scan-base-package=io.mosip.esignet.mock.integration,io.mosip.authentication.esignet.integration

```

Comment:
mosip.esignet.integration.scan-base-package=io.mosip.esignet.mock.integration

mosip.esignet.integration.binding-validator=BindingValidatorServiceImpl
mosip.esignet.integration.authenticator=MockAuthenticationService
mosip.esignet.integration.key-binder=MockKeyBindingWrapperService
mosip.esignet.integration.audit-plugin=LoggerAuditService
mosip.esignet.integration.vci-plugin=MockVCIssuancePlugin

As service in "countryA" will be integrated with mock-IDA, kindly update the above

Reply:
updated as above